### DISCUSSION TITLE
Help me recover my account fast

### DISCUSSION THREAD

**Discussion by DebiduttaPanigrahi** [community/community#119877] [Other Feature Feedback, Questions, & Ideas]
I have a different account which was my main account named DUTTA-78150 from debidutta.panigrahi@gmail.com as email id is automatically suspended without any reason please recover my account fast on that i have lots of project

**Comment by ebndev** [comment]
Hi @DebiduttaPanigrahi,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
